We force "daemonize no" in init script, so daemonize and pidfile does not
have any effect in redict.conf. It's init/rc system's job to daemonize
programs and handle pidfiles, if it needs it!

Applications often communicate with redict via unix socket, so it's
convenient to have it enabled by default.

--- a/redict.conf
+++ b/redict.conf
@@ -112,8 +112,8 @@
 # incoming connections. There is no default, so redict will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /run/redict.sock
-# unixsocketperm 700
+unixsocket /run/redict/redict.sock
+unixsocketperm 770
 
 # Close the connection after a client is idle for N seconds (0 to disable)
 timeout 0
@@ -241,11 +241,6 @@
 
 ################################# GENERAL #####################################
 
-# By default redict does not run as a daemon. Use 'yes' if you need it.
-# Note that redict will write a pid file in /var/run/redict.pid when daemonized.
-# When redict is supervised by upstart or systemd, this parameter has no impact.
-daemonize no
-
 # If you run redict from upstart or systemd, redict can interact with your
 # supervision tree. Options:
 #   supervised no      - no supervision interaction
@@ -264,20 +259,6 @@
 #
 # supervised auto
 
-# If a pid file is specified, redict writes it where specified at startup
-# and removes it at exit.
-#
-# When the server runs non daemonized, no pid file is created if none is
-# specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/redict.pid".
-#
-# Creating a pid file is best effort: if redict is not able to create it
-# nothing bad happens, the server will start and run normally.
-#
-# Note that on modern Linux systems "/run/redict.pid" is more conforming
-# and should be used instead.
-pidfile /var/run/redict_6379.pid
-
 # Specify the server verbosity level.
 # This can be one of:
 # debug (a lot of information, useful for development/testing)
@@ -289,7 +270,7 @@
 # Specify the log file name. Also the empty string can be used to force
 # redict to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile ""
+logfile /var/log/redict/redict.log
 
 # To enable logging to the system logger, just set 'syslog-enabled' to yes,
 # and optionally update the other syslog parameters to suit your needs.
@@ -441,7 +422,7 @@
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/redict
 
 ################################# REPLICATION #################################
 
